app/assets/javascripts/models/knockout_extensions.js.coffee

```diff
@@ -207,3 +207,19 @@ do (ko, $=jQuery) ->
     ko.bindingHandlers.text.update element, ->
       truncatedText
     return
+
+  ko.bindingHandlers.descriptionLinks =
+    init: (element, valueAccessor, allBindings, viewModel, bindingContext) ->
+      description = $($.parseHTML(bindingContext.$data.description))
+      newDescription = ''
+      for el in description
+        if el.nodeName == 'A' && el.host != window.location.host
+          $(el).attr('target', '_blank')
+          icon = "<i class=\"fa fa-external-link\" aria-hidden=\"true\"></i>"
```

Comment:
Rather than including new markup here the EUI (even though we don't use the actual library, we still try and follow the best practices) prefers using a class on the `<a>` tag. 

See [Typography](https://cdn.earthdata.nasa.gov/eui/latest/docs/eui/index.html#type) from the EUI documentation.

You can see the css used in EUI [here](https://cdn.earthdata.nasa.gov/eui/latest/stylesheets/application.css)